examples/function/asyncio-get/executorpool.py

```diff
@@ -0,0 +1,132 @@
+import asyncio
+import concurrent.futures
+import os
+from typing import AsyncIterable
+
+import aiorun
+import requests
+
+from pynumaflow import setup_logging
+from pynumaflow.function import Messages, Message, Datum, Metadata, UserDefinedFunctionServicer
+
+_LOGGER = setup_logging(__name__)
+
+
+class ExecutorPool:
```

Comment:
Is the intent to provide a convenience function? default executor pool would work just as well right?

Reply:
Yes, this was just to provide a wrapper around the basic functionalities of the executor. A default would work fine as well